cmd/api/routers/routers.go

```diff
@@ -0,0 +1,32 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package routers
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+// FIXME This will be taken from a shared pkg with sink based on the DB schema
+// Just for the first approach
+type resources struct {
+	Kind       string   `json:"kind"`
+	APIVersion string   `json:"apiVersion"`
+	Items      []string `json:"items"`
+}
+
+// GetAllResources responds with the list of resources of a specific type across all namespaces
+func GetAllResources(c *gin.Context) {
+	group := c.Param("group")
+	version := c.Param("version")
+	pluralResourceType := c.Param("resourceType")
+	response := resources{
+		// FIXME The plural of the resource type can be different of adding an `s` to the kind
+		Kind:       pluralResourceType[:len(pluralResourceType)-1],
+		APIVersion: fmt.Sprintf("%s/%s", group, version),
+		Items:      []string{"resource1", "resource2"},
+	}
+	c.IndentedJSON(http.StatusOK, response)
```

Comment:
While useful for development, `IndentedJSON` should not be used here in the final code. If you want to keep it this way for development, you should add a `FIXME` so we don't forget to change it later.

```suggestion
        // FIXME Replace with 'c.JSON(http.StatusOK, response)' when development is complete
	c.IndentedJSON(http.StatusOK, response)
```

See https://pkg.go.dev/github.com/gin-gonic/gin#Context.IndentedJSON for details.

Reply:
I've replaced IndentedJSON by JSON. It's short enough to be readable even if we are in a development phase. Also there are JSON parsers as browser plugins or we can use jq if we query the API from the command line.